src/lib/functions/scheduled.js

```diff
@@ -0,0 +1,78 @@
+const ansi2html = require('ansi2html')
+
+const { CLOCKWORK_USERAGENT } = require('../../utils')
+
+const { formatLambdaError } = require('./utils')
+
+const handleScheduledFunction = ({ error, request, response, result }) => {
+  const acceptsHtml = request.headers.accept && request.headers.accept.includes('text/html')
+  const paragraph = (text) => {
```

Comment:
NIT: I think you can move this out of this function as you don't need the outer context

Reply:
it needs `acceptsHtml`: https://github.com/netlify/cli/pull/3689/files/fa6f2d514dd00267573708e859b751a698f88a64#diff-4227587c310bc932a01198e05d935b060aa74a99364c8848af38478654d493beR12